pkg/util/globalerror/errors.go

```diff
@@ -52,6 +52,11 @@ const (
 	RequestRateLimited   ID = "tenant-max-request-rate"
 	IngestionRateLimited ID = "tenant-max-ingestion-rate"
 	TooManyHAClusters    ID = "tenant-too-many-ha-clusters"
+
+	OutOfBounds                 ID = "out-of-bounds"
```

Comment:
We want to address https://github.com/grafana/mimir/issues/1908. In the issue, Bryan suggests to translate it to something like "timestamp too old" and I agree. Here we could call it `sample-timestamp-too-old`